tests/route/test_static_route.py

```diff
@@ -164,10 +164,9 @@ def _check_routes():
                 return False
         return True
 
-    pytest_assert(
-        wait_until(60, 15, 0, _check_routes),
-        f"Route {prefix} advertisement state does not match expected 'removed={removed}' on all neighbors"
-    )
+    assert wait_until(60, 15, 0, _check_routes), (
+        "Failed to verify route redistribution: prefix '{}' not found in advertised routes of all BGP neighbors."
+    ).format(prefix)
```

Comment:
In the original summary, it prints out `removed`. It's better to keep the original summary content but with format not f.

`f"Route {prefix} advertisement state does not match expected 'removed={removed}' on all neighbors"`